solarforecastarbiter/validation/tasks.py

```diff
@@ -60,7 +62,10 @@ def validate_ghi(observation, values):
         _return_mask=True)
     ghi_clearsky_flag = validator.check_ghi_clearsky(values, clearsky['ghi'],
                                                      _return_mask=True)
-    return timestamp_flag, night_flag, ghi_limit_flag, ghi_clearsky_flag
+    cloudy_flag = validator.detect_clearsky_ghi(values, clearsky['ghi'],
+                                                _return_mask=True)
```

Comment:
I thought you'd want a try/except here instead of a warning in the function.

Reply:
I went back and forth on this. That is one option, but the rest of the functions in validator.py will not raise an error unless an invalid parameter for the algorith is chosen vs invalid data. 

Also, it's much easier to return a series of 0 in the function and have those converted to the flag vs having to pull in logic from quality_mapping.py to figure out what the flag should be in the except